README.md

```diff
@@ -144,41 +144,41 @@ None
 
 * If you can't execute the file directly after downloading it, try running `chmod u+x ./jitsi-meet-x86_64.AppImage`
 
-* Under wayland, screensharing is currently buggy:
+* Under Wayland, screensharing is currently buggy:
   * Sharing a full screen is not possible
-  * When trying to start screensharing under wayland, 2 permission popups will show up. First a pipewire based system selector, then a jitsi internal selector. Select an application window in the first selector and then the same in the jitsi internal selector. Sharing application windows works via this, sharing a full screen unfortunately not.
+  * When trying to start screensharing under Wayland, 2 permission popups will show up. First a PipeWire-based system selector, then a Jitsi internal selector. Select the application window in the first selector and click allow; then do the same in the Jitsi internal selector. This is how sharing application windows works, as sharing a full screen is unfortunately not possible.
 
-* On Ubuntu 22.04 and later, the AppImage will fail with a fuse error (as the AppImage uses `libfuse2`, while 22.04 comes with `libfuse3` by default):
+* On Ubuntu 22.04 and later, the AppImage will fail with a FUSE error (as the AppImage uses `libfuse2`, while 22.04 comes with `libfuse3` by default):
 
   ```
   dlopen(): error loading libfuse.so.2
   ```
 
-  To fix this, install libfuse2 as follows:
+  To fix this, install `libfuse2` as follows:
 
   ```
   sudo apt install libfuse2
   ```
 
 * On Ubuntu 24.04 and later, the AppImage will fail with a sandboxing error (`The SUID sandbox helper binary was found, but is not configured correctly...`)
-  This is due to AppArmor conflicting by restricting unprivileged user namespaces (https://github.com/jitsi/jitsi-meet-electron/issues/965,
+  This is due to an AppArmor conflict that restricts unprivileged user namespaces (jitsi/jitsi-meet-electron#965,
```

Comment:
I think the other link syntax is much more widespread. Please undo the change.

Reply:
Ok, I've made the change.